test/unit/environment.test.js

```diff
@@ -259,11 +257,38 @@ describe('the environment scraper', function () {
 
   describe('with symlinks', function () {
     const nmod = path.resolve(__dirname, '../helpers/node_modules')
+    const makeDir = (dirp) => {
```

Comment:
It seems weird to use both async/await and then/catch style, do you think we could update this function to be async/await for consistency?

Reply:
It should be something I can update--this already converted callback-style `fs` to `fs/promises`, which I could wrap in a try/catch block. Let's see.

Reply:
Should be updated!